public/app/plugins/datasource/prometheus/datasource.tsx

```diff
@@ -63,7 +62,7 @@ import {
 import { PrometheusVariableSupport } from './variables';
 
 const ANNOTATION_QUERY_STEP_DEFAULT = '60s';
-const GET_AND_POST_METADATA_ENDPOINTS = ['api/v1/query', 'api/v1/query_range', 'api/v1/series', 'api/v1/labels'];
+const POST_METADATA_ENDPOINTS = ['api/v1/query', 'api/v1/query_range', 'api/v1/series', 'api/v1/labels'];
```

Comment:
i'm not sure about the others, but should `api/v1/query` and `api/v1/query_range` be queries instead of resource calls?